C23_WMDE_Mobile_DE_13

moiikana opened the pull request:
Direct redirect 

https://phabricator.wikimedia.org/T353268

moiikana committed:
Archive C23_WMDE_Mobile_DE_12

moiikana committed:
Prepare campaign for C23_WMDE_Mobile_DE_13

- based on mobile de 12 ctrl
- close button on both softclose banners
- changed campaign day sentece in both banners

https://phabricator.wikimedia.org/T353268

Abban commented on banners/mobile/content/messages/DynamicCampaignTextOverwrite.de.ts:
```diff
@@ -1,19 +1,7 @@
-	'date-month-time-10': '{{day}} Oktober, um {{time}} Uhr',
-	'date-month-time-11': '{{day}} November, um {{time}} Uhr',
-	'date-month-time-12': '{{day}} Dezember, um {{time}} Uhr'
+	'campaign-day-only-n-days': 'Heute sind es nur noch {{days}} Tage bis zum Ende unserer Spendenkampagne.'
```
You shouldn't do it like this as the campaign days sentence already has set up for it, instead you should set the `urgencyMessageDaysLeft` to a higher number than there are days left in this campaign.

See https://github.com/wmde/fundraising-banners/blob/f6d0eb86c336971b995cabbc14aac650d09a9c9f/banners/desktop/banner_var.ts#L63 for an example

moiikana committed:
Implement VAR for C23_WMDE_Mobile_DE_13

- new button texts for variant submit
- direct redirect to the donation without address

https://phabricator.wikimedia.org/T353268

moiikana committed:
Integrate changes from 09 into Mikings design permanently

Abban committed:
Simplify the address type setter

This moves the functionality that sets the address type
based on payment type into a composable.

That means we can use the same form step controllers in
CTRL and VAR and no longer need 2 new ones.

Tickets:
- https://phabricator.wikimedia.org/T353268
- https://phabricator.wikimedia.org/T352973

Abban committed:
Merge pull request #324 from wmde/address-type-setter

Simplify the address type setter
